pscheduler-server/pscheduler-server/api-server/pschedulerapiserver/admin.py

```diff
@@ -111,3 +113,57 @@ def mtu_safe():
     except Exception as ex:
         log.exception()
         return error(str(ex))
+
+@application.route("/status", methods=['GET'])
+def get_status():
+    response = {}
+
+    response["time"] = str(pscheduler.datetime_as_iso8601(datetime.datetime.now()))
+
+    # query process table
+    services = {}
+    items = ["scheduler", "archiver", "ticker", "runner", "database"]
```

Comment:
For this short a list it probably doesn't matter, but in larger ones you'll want to use a hash so use of the `in` operator runs faster.  You can leave this as-is.